media-video/vidcutter/vidcutter-6.0.5.1-r2.ebuild

```diff
@@ -15,6 +15,7 @@ if [[ ${PV} == 9999 ]];then
 	EGIT_REPO_URI="https://github.com/ozmartian/vidcutter"
 else
 	SRC_URI="https://github.com/ozmartian/${PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
+	SRC_URI+=" https://github.com/ozmartian/${PN}/commit/1d88825feb5a73a50d019914ba9d0008562a58ce.patch -> ${P}-libmpv-api2.patch"
```

Comment:
Up to you because I bet it doesn't matter here, but for bits like this, I've recently started doing something like ${PN}-${HASH}-description.patch so we don't end up downloading it repeatedly if upstream make a release w/o it (branches or something).

Reply:
I usually do that as well (albeit with `${HASH::10}` for bit shorter names), but this one looked unlikely and it's not my package so I went for simple rather than add a HASH variable just to repeat it.